Source/Scene/Model.js

```diff
@@ -1388,7 +1429,35 @@ define([
         }
 
         var bufferView;
+        var accessors = model.gltf.accessors;
         var bufferViews = model.gltf.bufferViews;
+
+        var accessorsForBufferViewId = {};
+        for (var id in accessors) {
+            if (accessors.hasOwnProperty(id)) {
+                var accessor = accessors[id];
+                bufferViewId = accessor.bufferView;
+                // Accessors must be in byte offset order
+                if (!defined(accessorsForBufferViewId[bufferViewId])) {
+                    accessorsForBufferViewId[bufferViewId] = [accessor];
+                } else {
+                    var inserted = false;
```

Comment:
You might be able to cleanup this logic by adding all the accessors for each bufferview, then going by and sorting them with a functor that uses `byteOffset`.